makefiles/docker.inc.mk

```diff
@@ -69,6 +68,22 @@ export DOCKER_ENV_VARS += \
   UNDEF \
   #
 
+# List of all exported environment variables that shall be passed on to the
+# Docker container since they might have been set through the command line
+# and environment.
+# Theire origin cant be checked since they are often redefined or overriden
+# in in Makefile/Makefile.include, etc and theire origin is changed to file
```

Comment:
```suggestion
# Their origin cannot be checked since they are often redefined or overriden
# in Makefile/Makefile.include, etc. and their origin is changed to file
```